centralstore/redis_store.go

```diff
@@ -261,6 +279,7 @@ func (r *RedisBasicStore) ChangeTraceStatus(traceIDs []string, fromState, toStat
 	if toState == DecisionDrop {
 		traces, err := r.traces.getTraceStatuses(conn, traceIDs)
 		if err != nil {
+			r.errs <- err
```

Comment:
Hmmmm...do we really want to BOTH have a channel of errors and an error return? I would have thought we'd do one or the other.

Reply:
Oops, you are right. That's a leftover. I removed it